test/sqatt/src/Examples/DispatchProcess.hs

```diff
@@ -162,22 +162,22 @@ test12 = TxsExample
 
 test13 :: TxsExample
 test13 = TxsExample
-  { exampleName = "Process 12 - Unique ID (Wrong)"
+  { exampleName = "Process 12 - Unique ID (Purpose - Wrong)"
   , txsModelFiles = [txsFilePath specDir "DisPro12-unique-id"]
-  , txsCmdsFiles = [txsCmdPath exampDir "DispatchProcess_Stepper_12_Wrong"]
+  , txsCmdsFiles = [txsCmdPath exampDir "DispatchProcess_Tester_12_PurposeWrong"]
   , txsServerArgs = []
-  , sutExample = Nothing
-  , expectedResult = Fail
+  , sutExample = javaSUT
+  , expectedResult = Message "Goal wrong: halted"
```

Comment:
Now that this tests uses Test Purpose instead of Stepper, the test does not fail when Goal is not hit. It just reports that goal halted and test still reports as Pass.
Example:
```
TXSSERVER CMD >>  TESTER Model PurposeWrong Sut
TXSSERVER RSP <<  MACK Tester started
TXSSERVER RSP <<  PACK TESTER
TXSSERVER CMD >>  TEST 4
TXSSERVER RSP <<  MACK .....1:  IN:  Act    { { ( Job, [ JobData(82,"N22s",45,78) ] ) } }
TXSSERVER RSP <<  MACK .....2:  OUT: Act    { { ( Finish, [ JobOut(82,1,3) ] ) } }
TXSSERVER RSP <<  MACK .....3:  OUT: No Output (Quiescence)
TXSSERVER RSP <<  MACK Goals: 1
TXSSERVER RSP <<  MACK Goal wrong: halted
TXSSERVER RSP <<  MACK PASS
```